### ISSUE TITLE
ostree-repo-related tests should be decorated to run only on rhel7

### ISSUE THREAD

**Issue by rplevka** [SatelliteQE/robottelo#4040]
this applies to Hammer at least. Since hammer is a client, it is supposed to support `ostree` content-type even when installed on rhel6 (as it might be configured to talk to rhel7 satellite).

However, using this scenario on rhel6 satellite causes the scenario to fail as Rhel6 satellite does not recognize 'ostree' for a content type.

We should decorate these tests to run only on 'rhel7+'

**Comment by sghai** [comment]
Following flag is already set at all tests in UI. For api/cli, we had set it at  `setUpClass`
@skip_if_os('RHEL6')
 So this issue can be closed.

**Comment by sghai** [comment]
@rplevka : that's a negative test to check if download policy can be set for non-yum repo. On rhel6, user won't see `ostree` content-type as we are enabling it only for rhel7. 

So we don't need to decorate this test w/ ostree decorator.

**Comment by rplevka** [comment]
@sghai sorry, i pasted the wrong part of the stdout. I wanted to paste the following:

```
E                   AssertionError: "Validation failed: Download policy cannot be set for non-yum repositories" does not match "Failed to create Repository with data:
E                   {
E                     "checksum-type": null, 
E                     "content-type": "ostree", 
E                     "docker-upstream-name": null, 
E                     "download-policy": "on_demand", 
E                     "gpg-key": null, 
E                     "gpg-key-id": null, 
E                     "label": null, 
E                     "name": "mEiuUcbEkkovEpn", 
E                     "organization": null, 
E                     "organization-id": null, 
E                     "organization-label": null, 
E                     "product": null, 
E                     "product-id": "93", 
E                     "publish-via-http": "true", 
E                     "url": "http://inecas.fedorapeople.org/fakerepos/zoo3/"
E                   }
E                   Command "repository create" finished with return_code 64
E                   stderr contains following message:
E                   [ERROR 2016-11-22 05:27:27 Exception] Error: option '--content-type': value must be one of 'docker', 'file', 'yum', 'puppet'
```

**Comment by sghai** [comment]
Thanks @rplevka. That exception helps to understand the issue. 

Issue is that test is reading all REPO_TYPES from[ here](https://github.com/SatelliteQE/robottelo/issues/4040#issuecomment-270617213) and it includes `ostree`. Since ostree is not enabled on rhel6, so the exception is.

**Comment by sghai** [comment]
handled by #4183 #4182

**Comment by rochacbruno** [comment]
Fixed by #4183 and #4182
